tests/test_repositories.py

```diff
@@ -103,7 +95,7 @@ def test_create_virtual_repository_using_create_repo_fail_if_user_already_exists
     responses.add(

```

Comment:
Wait, all these tests about repositories have `if user already exists` in their name? :sweat_smile: